src/containers/my-cooperations/accept-cooperation-close/AcceptCooperationClosing.tsx

```diff
@@ -60,13 +60,13 @@ const AcceptCooperationClosing: React.FC<AcceptCooperationClosureProps> = ({
           </Typography>
           {t('cooperationDetailsPage.closingMessage2')}
           {message && (
-            <Box sx={styles.response}>
+            <Box component='span' sx={{ ...styles.response, display: 'block' }}>
```

Comment:
Can we move that to the `styles` file without breaking the layout?
```suggestion
            <Box component='span' sx={styles.response}>
```

Reply:
Changed this one, thanks for suggestion